cg/meta/deliver.py

```diff
@@ -210,6 +213,21 @@ def deliver_sample_files(
             f"There were {number_previously_linked_files} previously linked files and {number_linked_files_now} were linked for sample {sample_id}, case {case_id}"
         )
 
+        if self.delivery_type == Workflow.MICROSALT:
+            self.concatenate_fastqs(sample_directory=delivery_base, sample_name=sample_name)
+
+    def concatenate_fastqs(self, sample_directory: Path, sample_name: str):
+        if self.dry_run:
+            return
+        forward_out_path = Path(sample_directory, f"{sample_name}_R1.fastq.gz")
+        reverse_out_path = Path(sample_directory, f"{sample_name}_R2.fastq.gz")
```

Comment:
The old logic added the order date to the concatenated file name. I did it at first, but it looks so ugly. I suggest we skip it unless the customers actually want it (which we do not know)